src/include/simeng/arch/aarch64/Architecture.hh

```diff
@@ -15,6 +15,18 @@ namespace simeng {
 namespace arch {
 namespace aarch64 {
 
+/** Enum which holds AArch64 System register tags used by SimEng. */
+enum ARM64_SYSREG_TAGS : uint16_t {
+  DCZID_EL0 = 0,
```

Comment:
Could we change this to have the numbering be implicit

Reply:
Not possible for enums. What would be the benefit of making the numbering implicit?

Reply:
Implicity value allocation is possible, and also looks cleaner

Reply:
Sorry, I misunderstood what you said, I thought you meant do something similar to `systemRegisterMap_.size()`. Changing it

Reply:
done